poetry.lock

```diff
@@ -375,83 +393,115 @@ i18n = ["Babel (>=2.7)"]
 
 [[package]]
 name = "jsonschema"
-version = "4.17.3"
```

Comment:
Dev dependency `asynctest` for `python@3.7` was setting version `4.17.3` of this library, so no `ImportError` is triggered when running tests or in development environment.
However, when installing `ocpp` in a project `asynctest` is not installed, so `jsonschema` is set to `4.19.1`, causing the `ImportError` in production environment.

Reply:
I don't understand the relation between `asynctest` and `jsonschema`. Can you elaborate?
When dropping support for Python 3.7,  the dependency `asynctest` becomes obsolete and can be dropped.